### DISCUSSION TITLE
How can I output the result of Duckdb query into a file using python

### DISCUSSION THREAD

**Discussion by woshimajintao** [duckdb/duckdb#9420] [Q&A]
How can I output the result of Duckdb query into a file using python?

import duckdb
duckdb.sql("""select * from tpcds_queries();""")

**Comment by szarnyasg** [comment]
Create a Python loop:
```
import duckdb
for i in range(1, 100):
  duckdb.sql(f"""copy (select query from tpcds_queries() where query_nr = {i}) to 'q{i:02d}.sql' (header false, quote '')""")
```

**Reply by woshimajintao** [reply]
Thanks.
I used the code:

import pandas as pd
import duckdb
duckdb.sql("""select * from tpcds_queries();""").write_csv('output1.csv')  

import pandas as pd
chunksize = 1
path = 'C:\\Users\\Jintao1999\\Desktop\\python\\output1.csv'
for i,chunk in enumerate(pd.read_csv(path, chunksize=chunksize)):
    chunk.to_csv('D:\\Code\\duckdb_queries\\{}.sql'.format(i+1), index=False,encoding='ANSI' )
